apps/web/components/NavBar/NavMenu.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { PersonIcon, MeetingIcon, SeatsIcon, EquipmentIcon } from "@repo/ui/public";
+import { usePathname } from "next/navigation";
+import Link from "next/link";
+import { clsx } from "clsx";
+
+const NAV_ITEMS = [
+  { href: "/dashboard", name: "대시보드", icon: PersonIcon },
+  { href: "/meeting-room", name: "회의실", icon: MeetingIcon },
+  { href: "/seats", name: "좌석", icon: SeatsIcon },
+  { href: "/equipment", name: "장비", icon: EquipmentIcon },
+];
```

Comment:
constants/routes.ts 경로로 파일을 하나 구성해서 route를 상수로 관리해도 좋을거같아
```js
// routes.ts

export const ROUTES = {
  DASHBOARD: { href: "/dashboard", name: "대시보드" },
  MEETING_ROOM: { href: "/meeting-room", name: "회의실" },
  SEATS: { href: "/seats", name: "좌석" },
  EQUIPMENT: { href: "/equipment", name: "장비" },
};
```
파일 구성한다면 이런식으로 작성될거같고
```suggestion
const NAV_ITEMS = [
  { href: ROUTES.DASHBOARD.href, name: ROUTES.DASHBOARD.name, icon: PersonIcon },
  { href: ROUTES.MEETING_ROOM.href, name: ROUTES.MEETING_ROOM.name, icon: MeetingIcon },
  { href: ROUTES.SEATS.href, name: ROUTES.SEATS.name, icon: SeatsIcon },
  { href: ROUTES.EQUIPMENT.href, name: ROUTES.EQUIPMENT.name, icon: EquipmentIcon },
];
```

이렇게 한다면 관리나 수정이 더 편해질거같아!

Reply:
type이 붙는다면 as const 추가 정도...?

Reply:
음 혹시 이 route라는 상수가 확장성이 있어서 그러는걸까? 확장성 있게 쓰인다면 어디에서 쓰인다고 생각되는지 알려줄 수 있어? GNB에서만 쓰이는거면 알려준 방법이 유지/관리가 더 복잡할 것 같아서

Reply:
@cindycho0423 GNB안에서만 route가 관리된다면 굳이 분리 안해도 되겠다! 다른곳에서 쓰이는건 생각 안해봤어